app/assets/stylesheets/application.scss

```diff
@@ -0,0 +1,52 @@
+/*
+ * This is a manifest file that'll be compiled into application.css, which will include all the files
+ * listed below.
+ *
+ * Any CSS and SCSS file within this directory, lib/assets/stylesheets, or any plugin's
+ * vendor/assets/stylesheets directory can be referenced here using a relative path.
+ *
+ * You're free to add application-wide styles to this file and they'll appear at the bottom of the
+ * compiled file so the styles you add here take precedence over styles defined in any other CSS/SCSS
+ * files in this directory. Styles in this file should be added after the last require_* statement.
+ * It is generally better to create a new file per style scope.
+ *
+ *= require_tree .
+ *= require_self
+ */
+
+ @import "bootstrap/scss/bootstrap";
+ @import url('https://fonts.googleapis.com/css?family=Numans');
+ @import url('https://fonts.googleapis.com/css2?family=Open+Sans:wght@400;700&family=Poppins:wght@200;400;600&family=Spartan:wght@400;500;700&display=swap');
+ @import url('https://fonts.googleapis.com/css2?family=Nunito:wght@600&display=swap');
+
+
+html,body{
+background: rgb(6,61,140);
+background: linear-gradient(0deg, rgba(6,61,140,1) 0%, rgba(91,201,246,1) 100%);
+background-size: cover;
+background-repeat: no-repeat;
+height: 100%;
+font-family: 'Nunito', sans-serif;
+}
+.header .navbar  {
+    background-color: transparent !important;
+}
+
+.navbar-logo{
+    font-family: 'Nunito', sans-serif;
+    font-size: 30px;
+    letter-spacing: 2px;
+    word-spacing: 2px;
+    color: #e5e5e5;
+    font-weight: 700;
+    text-decoration: none;
+    font-style: normal;
+    font-variant: normal;
+    text-transform: capitalize;  
+}
+.container{
+align-content: center;
+width: 380px;
+border-radius: 10px;
+overflow: hidden;
```

Comment:
Same for this code, and sort it, remeber that can doing this using ` ctrl+shift+p` and select the function sort lines ascending in visual studio code